docs/source/models/cause.rst

```diff
@@ -558,6 +558,53 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say there
+  were 1 million prevalent cases of diarrhea over the course of 2017, and each
```

Comment:
Are you referring to the period prevalence here as opposed to a point prevalence?  If so, we should probably clarify that because we usually mean point prevalence.

Reply:
Yes, I am. I could also have said incidence; do you think that would be clearer @NathanielBlairStahn ?

Reply:
This is an interesting point! 

In GBD, incidence of diarrheal diseases >>> (point) prevalence of diarrheal diseases because of the high remission.

I'm not sure what the best approach to discussing this in this section will be. FYI Nathaniel, we're in coworking now and will discuss.